### DISCUSSION TITLE
404 at https://codeload.github.com/twbs/bootstrap/zip/v5.0.0

### DISCUSSION THREAD

**Discussion by IntegralWorks** [twbs/bootstrap#31215] [General]
https://codeload.github.com/twbs/bootstrap/zip/v5.0.0 returns a simple "404: Not Found".

**Comment by XhmikosR** [comment]
Yeah, because it's not 5.0.0, it's 5.0.0-alpha1 (https://codeload.github.com/twbs/bootstrap/zip/v5.0.0-alpha1). Where did you get that link from?
